src/plugins/intel_cpu/src/transformations/cpu_opset/common/pass/move_fc_reshape_to_weights.cpp

```diff
@@ -0,0 +1,102 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/cpu_opset/common/op/fully_connected.hpp"
+#include "move_fc_reshape_to_weights.hpp"
+#include <transformations/utils/utils.hpp>
+#include <openvino/pass/pattern/op/wrap_type.hpp>
+#include <openvino/pass/pattern/op/or.hpp>
+#include <openvino/opsets/opset1.hpp>
+
+#include "itt.hpp"
+
+ov::intel_cpu::MoveFCReshapeToWeights::MoveFCReshapeToWeights() {
+    MATCHER_SCOPE(MoveFCReshapeToWeights);
+    using namespace ov::pass::pattern;
+    auto weights_m = wrap_type<ov::opset1::Constant>(consumers_count(1));
```

Comment:
the current recommendation is to use the concrete operation versions, e.g.
ov::op::v0::Constant instead of ov::opset1::Constant due to CVS-91401 
and do not include opset headers (<openvino/opsets/opset1.hpp>)

In short, if we include the opset header, it adds ~5mb to obj file size, there are lots of transformations (N) and every transformation corresponds to its own obj file (N * 5mb), so the final lib size can exceed 4gb limit on Win platform in debug configuration

Reply:
Thank you for the explanation. I refactored the both affected by this PR transformations to match your suggestions